model/flow/identity.go

```diff
@@ -167,8 +186,7 @@ type encodableIdentitySkeleton struct {
 
 type encodableIdentity struct {
 	encodableIdentitySkeleton
-	Weight  uint64
-	Ejected bool
+	ParticipationStatus string
```

Comment:
Probably not edit, but read. We also store the `Role` as a string when encoding to JSON rather than something smaller and it makes working with snapshot files etc. easier.